### PR TITLE
snowpack 3.8.8

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by cho-m** [comment]
Need to check on ARM. There are no prebuilds for `darwin-arm64`, but may be okay if prebuilds are only a fast-path. However, a nested dependency `rollup --> fsevents` is using the x86_64-specific version rather than the universal-version (which would be fixed by thinning binary).
```
==> FAILED
snowpack:
  * Binaries built for a non-native architecture were installed into snowpack's prefix.
    The offending files are:
      /opt/homebrew/Cellar/snowpack/3.8.8/libexec/lib/node_modules/snowpack/node_modules/bufferutil/prebuilds/darwin-x64/node.napi.node	(x86_64)
      /opt/homebrew/Cellar/snowpack/3.8.8/libexec/lib/node_modules/snowpack/node_modules/rollup/node_modules/fsevents/fsevents.node	(x86_64)
      /opt/homebrew/Cellar/snowpack/3.8.8/libexec/lib/node_modules/snowpack/node_modules/utf-8-validate/prebuilds/darwin-x64/node.napi.node	(x86_64)
Error: 1 problem in 1 formula detected
```

**Comment by carlocab** [comment]
Just delete them, I think.

File: Formula/snowpack.rb [hunk 1]
```diff
@@ -19,7 +19,19 @@ class Snowpack < Formula
...
+    bin.install_symlink Dir[libexec/"bin/*"]
+
+    # Remove incompatible pre-built binaries
+    os = "darwin"
+    on_linux { os = "linux" }
+    arch = Hardware::CPU.arm? ? "arm64" : "x64"
+    libexec.glob("lib/node_modules/snowpack/node_modules/{bufferutil,utf-8-validate}/prebuilds/*")
+           .each { |dir| dir.rmtree if dir.basename.to_s != "#{os}-#{arch}" }
+    # `rollup` < 2.38.3 uses x86_64-specific `fsevents`. Can remove when `rollup` is updated.
+    (libexec/"lib/node_modules/snowpack/node_modules/rollup/node_modules/fsevents").rmtree if Hardware::CPU.arm?
```
Comment (cho-m) [inline Formula/snowpack.rb:31]: There are a few different style alternatives to perform the deletion. Can be modified if any preference.

Comment (chenrui333) [reply Formula/snowpack.rb:31]: we can also revisit this after `bufferutil`/`utf-8-validate` doing the universal binary.

Comment (Tabbbbs) [reply Formula/snowpack.rb:31]: I just need to link without my hit.

File: Formula/snowpack.rb [hunk 2]
```diff
@@ -19,7 +19,19 @@ class Snowpack < Formula
 
   def install
     system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
+    bin.install_symlink Dir[libexec/"bin/*"]
+
+    # Remove incompatible pre-built binaries
+    os = "darwin"
+    on_linux { os = "linux" }
```
Comment (carlocab) [inline Formula/snowpack.rb:26]:
```suggestion
    os = OS.mac? ? "darwin" : "linux"
```
I think we've decided it's okay to use `OS.mac?` in `install` methods now, so let's make this a bit less ugly.

Comment (chenrui333) [reply Formula/snowpack.rb]: 👍

**Pull request closed without merging**